examples/tensorflow/iris-classifier/predictor.py

```diff
@@ -0,0 +1,13 @@
+# WARNING: you are on the master branch, please refer to the examples on the branch that matches your `cortex version`
+
+labels = ["iris-setosa", "iris-versicolor", "iris-virginica"]
```

Comment:
Should this be in `init()`? I'm not sure that it should, just wanted you to consider